app/views/recipies/public_recipies.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Public recipes</h2>
+<ul class="public_recipes_list">
+  <% @recipies.each do |recipe| %>
+    <li class="public_recipe">
+<<<<<<< HEAD
+      <h3><%= link_to recipe.name,"/recipies/#{recipe.id}" %></h3>
```

Comment:
I think here we could use the rails route helpers instead of writing it manually, but wait for me because I need to change the routes name